js/ui/text_box/ui.text_editor.mask.strategy.default.js

```diff
@@ -113,6 +117,10 @@ class DefaultMaskStrategy extends BaseMaskStrategy {
         this._backspaceHandler(event);
     }
 
+    _isAutoFill(event) {
+        return event && event.originalEvent && !event.originalEvent.inputType;
```

Comment:
Is this a reliable condition? Why 'inputType' is absent on iOS? Is this specific to iOS only?